catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
```

Comment:
We should follow [The Unofficial Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) or at the very least use `set -euo pipefail`